core/geometry/src/geometry3d/Matrix3d.ts

```diff
@@ -2525,86 +2552,105 @@ export class Matrix3d implements BeJSONFunctions {
   public get hasCachedInverse(): boolean {
     return this.inverseState === InverseMatrixState.inverseStored && this.inverseCoffs !== undefined;
   }
-  /** Test if the below diagonal entries are all nearly zero */
+  /** Test if the below diagonal entries (3,6,7) are all nearly zero */
   public get isUpperTriangular(): boolean {
     const sumAll = this.sumSquares();
     const sumLow = Geometry.hypotenuseSquaredXYZ(this.coffs[3], this.coffs[6], this.coffs[7]);
     return Math.sqrt(sumLow) <= Geometry.smallAngleRadians * (1.0 + Math.sqrt(sumAll));
   }
-  /** If the matrix is diagonal and all diagonals are within tolerance, return the first diagonal.  Otherwise return undefined.
+  /** Test if the above diagonal entries (1,2,5) are all nearly zero */
+  public get isLowerTriangular(): boolean {
+    const sumAll = this.sumSquares();
+    const sumLow = Geometry.hypotenuseSquaredXYZ(this.coffs[1], this.coffs[2], this.coffs[75]);
+    return Math.sqrt(sumLow) <= Geometry.smallAngleRadians * (1.0 + Math.sqrt(sumAll));
+  }
+  /**
+   * If the matrix is diagonal and all diagonals are almost equal, return the first diagonal (entry 0
+   * which is same as entry 4 and 8). Otherwise return `undefined`.
    */
   public sameDiagonalScale(): number | undefined {
     const sumAll = this.sumSquares();
     const sumDiagonal = this.sumDiagonalSquares();
     const sumOff = Math.abs(sumAll - sumDiagonal);
-    if (Math.sqrt(sumOff) <= Geometry.smallAngleRadians * (1.0 + Math.sqrt(sumAll))
-      && Geometry.isSameCoordinate(this.coffs[0], this.coffs[4]) && Geometry.isSameCoordinate(this.coffs[0], this.coffs[8]))
+    if (Math.sqrt(sumOff) <=
+      Geometry.smallAngleRadians * (1.0 + Math.sqrt(sumAll))
+      && Geometry.isSameCoordinate(this.coffs[0], this.coffs[4])
+      && Geometry.isSameCoordinate(this.coffs[0], this.coffs[8])
+    )
       return this.coffs[0];
     return undefined;
   }
-  /** Sum of squared differences between symmetric pairs */
+  /** Sum of squared differences between symmetric pairs (entry 1 and 3 - 2 and 6 - 5 and 7) */
```

Comment:
The dashes confused me as minus signs. Consider a new sentence: "Symmetric pairs have indices (1,3), (2,6), and (5,7)."

Reply:
will fix